### PR TITLE
Wuhu Island (Created by Tizona, Ported By Me) KOTH

### PR DESCRIPTION
**Pull request by JackJack33**
A literal monstrosity of a map. This contains 10 control points and only 2 teams. MADE FOR LARGE EVENTS! DO NOT ADD TO ROTATION! (Easter eggs included!)

### CODE REVIEW

**Comment by JackJack33** [comment]
Fixed brackets!

File: KOTH/Wuhu/map.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
+{
+	"name": "Wuhu Island",
+	"authors": ["Tizona (Ported by JackJack33)"],
```
Comment (BennyDoesTheStuff) [inline KOTH/Wuhu/map.json]: You might want to remove the whole "Ported by JackJack33" message.

File: KOTH/Wuhu/map.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+		},
+		{
+			"id": "red",
+			"name": "Red",
+			"color": "red",
+			"max": 100,
+			"min": 1
+		}
+	],
+
```
Comment (BennyDoesTheStuff) [inline KOTH/Wuhu/map.json]: Remove this blank line.

File: KOTH/Wuhu/map.json [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+	"spawns": [
+		{ "teams": ["spectators"], "x": -1222.5, "y": 46, "z": 459.5, "yaw": 0 },
```
Comment (BennyDoesTheStuff) [inline KOTH/Wuhu/map.json]: For all the spawns, it's better if you instead did "-1222.5, 46, 459.5". If the yaw is 0, you don't have to add it.

File: KOTH/Wuhu/map.json [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+		}
+	],
+	"itemremove": [
+		"stone sword", "arrow", "chainmail chestplate", "leather boots",
+		"dirt", "golden carrot", "leather leggings"
+	],
+	"filters": [
+		{ "type": "enter", "evaluate": "deny", "teams": ["blue"], "regions": ["red-spawn-protection"], "message": "&cYou may not enter the enemy spawn."},
+		{ "type": "enter", "evaluate": "deny", "teams": ["red"], "regions": ["blue-spawn-protection"], "message": "&cYou may not enter the enemy spawn."}
+
```
Comment (BennyDoesTheStuff) [inline KOTH/Wuhu/map.json]: Another empty line you might want to remove.

**Comment by JackJack33** [comment]
Fixed unecessary things

**Pull request merged**